src/components/SUE/report/SueReportServices.tsx

```diff
@@ -14,7 +15,24 @@ export const SueReportServices = ({
   serviceCode: string;
   setServiceCode: any;
 }) => {
-  const { data } = useQuery([QUERY_TYPES.SUE.SERVICES], () => getQuery(QUERY_TYPES.SUE.SERVICES)());
+  const [loading, setLoading] = useState(true);
+
+  const { data, isLoading, refetch } = useQuery([QUERY_TYPES.SUE.SERVICES], () =>
+    getQuery(QUERY_TYPES.SUE.SERVICES)()
+  );
```

Comment:
i guess it should be possible with passing a config with no cache time (0 ms) to always load new data? could this be possible to not need the workaround with refetching?